### PR TITLE
chore(swagger): include help links

### PR DESCRIPTION
**Pull request by parano**
Screenshot 2022-08-19 at 20 21 24

### CODE REVIEW

File: examples/quickstart/service.py
Comment (parano) [inline examples/quickstart/service.py:12]: This helps with showing an example in the swagger page:
Screen Shot 2022-08-19 at 3 11 05 PM

File: setup.cfg
Comment (aarnphm) [inline setup.cfg:96]: This is to fix with newer version of setuptools that doesn't add package_data automatically.

File: setup.cfg
Comment (aarnphm) [inline setup.cfg:76]: No need to specify it here since we already did under `pyproject.toml`.

File: bentoml/_internal/server/static_content/index.css
Comment (aarnphm) [inline bentoml/_internal/server/static_content/index.css:1]: minified bundle

**Review by aarnphm — APPROVED** [review]

**Review by aarnphm — APPROVED** [review]

File: bentoml/_internal/bento/build_config.py
Comment (parano) [inline bentoml/_internal/bento/build_config.py:486]: @aarnphm let's wait for another review if you made changes to the PR

File: bentoml/_internal/bento/build_config.py
Comment (parano) [inline bentoml/_internal/bento/build_config.py:528]: why change the indentation here?

File: bentoml/_internal/server/static_content/index.css
Comment (parano) [inline bentoml/_internal/server/static_content/index.css:1]: @aarnphm let's revert this change back... you should never check in minified bundle like thihs

File: setup.cfg
Comment (parano) [inline setup.cfg:95]: let's put these type of changes to a separate PR and test separately next time @aarnphm

File: docs/source/conf.py
Comment (parano) [inline docs/source/conf.py:3]: @aarnphm same here. let's revert them for now and submit a separate PR for review

File: bentoml/_internal/bento/build_config.py
Comment (aarnphm) [reply bentoml/_internal/bento/build_config.py:486]: sg

**Pull request merged**
